tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -153,6 +157,14 @@ spec:
                   --rh-push $(params.rhPush) | tee "/tmp/output"
                 # The rh-push-to-external-registry e2e test depends on this line being in the task log
                 IMAGEID=$(awk '/The image id is/{print $NF}' /tmp/output)
+
+                # Remove the new image tags from all previous images
```

Comment:
I think it makes sense at this point to just export PYXIS_CERT_PATH and PYXIS_KEY_PATH instead of passing them twice

Reply:
I actually kind of like this approach of only setting it for the two commands - that means no other commands will see it which seems safer in principle. But if you have a strong opinion on setting it for the whole script, I can change it.